elements/modal.js

```diff
@@ -121,15 +122,17 @@ const prefix = css`
 module.exports = createWidget
 
 function createWidget () {
+  let isCopied = false
```

Comment:
i wonder if this is a little over engineered, couldn't the flow also just be `open modal -> not copied. copy -> copied. close modal -> reset state`